script.js

```diff
@@ -128,18 +160,110 @@ const footer = {
     template: `The software is provided by <a href="https://webix.com">https://webix.com</a>. All rights reserved &copy`
 }
 
+const userList = {
+    rows: [
+        {
+            view: "toolbar",
+            elements: [
+                {
+                    view: "text",
+                    id: "title_filter_input",
+                    placeholder: "Country name",
+                    on: {
+                        onTimedKeyPress: function () {
+                            const value = this.getValue().toLowerCase();
+                            $$("user_list").filter(function (obj) {
+                                return obj.country.toLowerCase().indexOf(value) !== -1;
```

Comment:
I'm confused @Zwillinge 
Should it filter only by country? The task is "Enable List filtering via a Text input"

Reply:
it should be filtered by name or by both name and country (visible values)

Reply:
Added additional filter for names.